### ISSUE TITLE
Move active development to 2.4.next branch

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#1582]
Right now we have the 2.4 branch with tags when a 2.4.x version is released. As a result not many users will run a "stable" 2.4.x version but rather some version in between 2.4.x and 2.4.x+1. 

By moving the active development to a 2.4.next branch and only merging this branch with the 2.4 branch when a 2.4.x version is released running the "bleeding edge" version will be an active choice.

**Comment by adulau** [comment]
It's indeed a valid point but the update procedure should be a git fetch then a checkout with the tagged version. https://github.com/MISP/MISP/blob/2.4/INSTALL/UPDATE.txt

The issue of having a bleeding edge .next branch IMHO is the contribution and pull request that might criss-cross a bit between the branches.

**Comment by RichieB2B** [comment]
Good points. As long as it's a conscience choice I'm happy with it either way.

**Comment by iglocska** [comment]
For sure, the idea is basically:

Anything that is a hotfix of a minor, mostly cosmetic feature can go directly to head (once it has been tested and approved).

Anything that changes functionality, carries the risk of breaking things should go to a feature branch. Before a new release is made the feature branches are consolidated into a point release branch (such as 2.4.53) and tested before release. 

Once that gets the OK it is merged into 2.4 and 2.4 gets a new tag. For anyone wanting to stay super safe, using a tagged version is the solution, but I'd consider anything that is on the 2.4 branch as stable (of course that doesn't mean that issues won't be found in it occasionally, though this is just as true for tagged releases). 

Hope it makes sense!
